packages/avatar/Avatar.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Skeleton from 'react-loading-skeleton';
+import Img from 'react-image';
+import { useEffectAsync } from '@availity/hooks';
+import { avSettingsApi } from '@availity/api-axios';
+import get from 'lodash.get';
```

Comment:
I don't think this was included in the deps.

Reply:
ec9883842bfdb01dc396cae2bde3c9d2f36dc4bb